editor/src/clj/editor/render_target.clj

```diff
@@ -0,0 +1,129 @@
+;; Copyright 2020-2023 The Defold Foundation
+;; Copyright 2014-2020 King
+;; Copyright 2009-2014 Ragnar Svensson, Christian Murray
+;; Licensed under the Defold License version 1.0 (the "License"); you may not use
+;; this file except in compliance with the License.
+;;
+;; You may obtain a copy of the License, together with FAQs at
+;; https://www.defold.com/license
+;;
+;; Unless required by applicable law or agreed to in writing, software distributed
+;; under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+;; CONDITIONS OF ANY KIND, either express or implied. See the License for the
+;; specific language governing permissions and limitations under the License.
+
+(ns editor.render-target
+  (:require [editor.build-target :as bt]
+            [dynamo.graph :as g]
+            [editor.graph-util :as gu]
+            [editor.gl.texture :as texture]
+            [editor.protobuf :as protobuf]
+            [editor.protobuf-forms :as protobuf-forms]
+            [editor.resource-node :as resource-node]
+            [editor.workspace :as workspace])
+  (:import [com.dynamo.graphics.proto Graphics$TextureImage$Type Graphics$TextureImage$TextureFormat]
+           [com.dynamo.render.proto RenderTarget$RenderTargetDesc]))
+
+(def ^:const atlas-icon "icons/32/Icons_13-Atlas.png")
+
+(defn- set-form-op [{:keys [node-id]} [property] value]
+  (g/set-property! node-id property value))
+
+(defn- clear-form-op [{:keys [node-id]} [property]]
+  (g/clear-property! node-id property))
+
+(def form-data
+  {:navigation false
+   :sections
+   [{:title "Render Target"
+     :fields [{:path [:color-attachments]
+               :label "Color Attachments"
+               :type :table
+               :columns [{:path [:width]
+                          :label "width"
+                          :type :number}
+                         {:path [:height]
+                          :label "height"
+                          :type :number}
+                         {:path [:format]
+                          :label "format"
+                          :type :choicebox
+                          :options (protobuf-forms/make-options (protobuf/enum-values Graphics$TextureImage$TextureFormat))
+                          :default (ffirst (protobuf/enum-values Graphics$TextureImage$TextureFormat))}]}
+              {:path [:depth-stencil-attachment-width]
+               :label "Depth/Stencil Width"
+               :type :number}
+              {:path [:depth-stencil-attachment-height]
+               :label "Depth/Stencil Height"
+               :type :number}
+              {:path [:depth-stencil-attachment-texture-storage]
+               :label "Depth Buffer Texture Storage"
+               :type :boolean}]}]})
```

Comment:
This doesn't seem to match the updated structure of `RenderTarget$RenderTargetDesc`?

Also, add line breaks between top-level forms.

Reply:
Oh, never mind. I just got confused... 😵‍💫
This seems fine after all.